src/admin-views/base_admin_panel.php

```diff
@@ -1,35 +1,39 @@
+<?php
+	$total_tickets = Tribe__Tickets__Tickets_Handler::instance()->get_total_event_capacity( $post_id );
+	$container_class = 'tribe_sectionheader ticket_list_container';
+	$container_class .= ( empty( $total_tickets ) ) ? ' _blank' : '' ;
```

Comment:
Whatever is easiest. It was not clear to me if this should behave differently when an unlimited stock is in place. If it should behave the same as when there is no stock, `empty()` works pretty good.